README.md

```diff
@@ -95,6 +98,13 @@ This is an example of `appconfig.json`:
 }
 ```
 
+# Next Steps
+There are multiple ways to deploy your extension to Kyma:
+
+* Automatic deployment via the "Extension App" UI inside "Foundational Services" in SAP Field Service <https://docs.coresystems.net/shell/extension-apps-management.html>
+* Automatic deployment via FSM API <https://github.com/SAP-samples/fsm-extension-sample/blob/master/docs/APIGuide.md>
+* Manual deployment via helm <https://kyma-project.io/docs/#installation-use-helm>.
```

Comment:
Some additional parameters (required by the extension itself) need to be provided in helm command
helm install ./artifacts/helm/{application name} --name {application name} --namespace {kyma namespace} **--set kyma.api.enabled=true --set kyma.version={kyma version}** --tls
And these parameters are not mentioned in the Kyma guide

Reply:
Simon and I found the optional Helm chart part distracting when you want to follow the main story, which would be the installation via the Extension App UI. Still I think the work should not be lost on the helm chart deployment. How about creating a separate guide for this inside this git repo and we link it here? Btw. after you did the helm chart deployment you would need to use the manual registration to get it to fsm?!

Reply:
Hi @tm4g1x 
It is a good point, we have two options:
1. Remove the manual installation via helm CLI part totally from the doc, since it is distracting.
2. Or create a new doc in the /docs folder
For me, I prefer the first one, since after 2008 release, people all are supposed to use the convenience extension management UI to deploy the extension ..

Reply:
@MingZhang-PS Ok, I will remove it entirely. Should we also update the metadata and appconfig.json helm chart entries?

![image](https://user-images.githubusercontent.com/24810353/88892771-a4579100-d245-11ea-92d0-8695965921c4.png)

Reply:
No, please keep them.
I just mean to remove description about  "manual install via helm CLI". 
Helm chart artifacts are still valid, they will be used even in extension app auto install case

Reply:
@MingZhang-PS removed all parts, where helm was mentioned. Thoguh the generator will still create folder structures and so on for it, but may we leave that part in?

Reply:
Hi @tm4g1x 
Let me make it to be more clear. All statement of which prefixed with "Optional" can be removed, since they are doing some manual work in CLI which is supposed to be replace by extension management UI after 2008.
And for other words, they shall be kept. Since the helm chart deployment type is a feature to be supported.
Could you please revert this commit and only remove the words in "[Optional]" lines?

Reply:
@MingZhang-PS done